src/profile/webid.test.ts

```diff
@@ -425,21 +457,24 @@ const mockProfileDoc = (
 
 describe("getPodUrlAll", () => {
   it("does not use the provided fetch to dereference the WebID", async () => {
-    const mockedFetcher = jest.requireMock("cross-fetch") as {
+    const { fetch: mockedFetcher } = jest.requireMock("../fetcher") as {
```

Comment:
Can we mock `@inrupt/universal-fetch` instead of `../fetcher` (here and in all the tests that follow)

Reply:
See the comment about the `fetcher` file